_posts/2019-08-26-how-to-use-gruntwork-service-catalog.adoc

```diff
@@ -1,14 +1,27 @@
 ---
-categories: Gruntwork
-image: /assets/img/guides/service-catalog/grunty-blocks.png
+categories: Foundations
+image: /assets/img/guides/service-catalog/grunty-grid.png
```

Comment:
I have changed images for all current posts in on-going work

Reply:
Changed them in what way?

Reply:
I switched them with the images Sam uploaded to Zeplin

Reply:
This is the thumbnail for this specific deployment guide though, not a generic image. Are we changing those too?

Reply:
Nah, I was just getting a better image since I was working on the images aligning anyway and the one before was basically a gigantic placeholder and zeplin happened to have one example with a grunty thumbnail, so I got that one for this post. 

We'll have to solve this conflict either on the other PR or this one. Which one do you prefer?

<img width="821" alt="Screen Shot 2019-09-17 at 11 53 08" src="https://user-images.githubusercontent.com/1516418/65035810-09620b00-d942-11e9-98da-72105cfd6aa1.png">
<img width="882" alt="Screen Shot 2019-09-17 at 11 45 19" src="https://user-images.githubusercontent.com/1516418/65035814-0bc46500-d942-11e9-94a8-f1ffddec315c.png">

Reply:
Oh, I gotcha. I don't have a strong preference, but I think the one on the purple background is our old logo image, and the one on the grid with arms folded is the newer one, so perhaps go with the newer branding?

Reply:
(You can pick this just verbally, and I'll fix it on the other PR)

Reply:
ok! that makes sense 👍

Reply:
Roger. OK, let's go with the newer Grunty then.